src/abstractdataframe/reshape.jl

```diff
@@ -399,3 +399,96 @@ function CategoricalArrays.CategoricalArray(v::RepeatedVector)
     res.refs = repeat(res.refs, inner = [v.inner], outer = [v.outer])
     res
 end
+
+
+Base.transpose(::AbstractDataFrame, args...; kwargs...) =
+    MethodError("`transpose` not defined for `AbstractDataFrame`s. Try `permutedims` instead")
+    # ↑ is 94 char, is that OK? (there are a couple other lines in this file that go over)
```

Comment:
Oh - and I'm not 100% clear on the character limit... there are a couple of places in this file that go over - was that comment just about the docstring?

Reply:
Here are the style guidelines: https://github.com/JuliaData/DataFrames.jl/blob/master/CONTRIBUTING.md#style-guidelines

(so in general it should be everywhere, but in practice we are not 100% consistent as sometimes keeping narrow lines looks ugly and this package is quite old and had many contributors :smile:)

Reply:
94 is OK I think in this case